### DISCUSSION TITLE
cannot access Security Onion on browser after installation.

### DISCUSSION THREAD

**Discussion by AVINASH7799** [Security-Onion-Solutions/securityonion#12301] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

12GB

### Storage for /

200GB

### Storage for /nsm

12

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

i guess the installation went well. the screen said access security onion through https://soavinash. ("soavinash" is the given name). but upon accessing the link, the browser response is as shown in the attachment. does this mean something is wrong with installation process? please help and thanks in advance.
![Screenshot (24)](https://github.com/Security-Onion-Solutions/securityonion/assets/64453627/f62c0eee-8516-48cd-92cb-3ff12a4aaf4a)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by Acewiza** [comment]
Security Onion didn't show that page.  Guessing networking numbers not well organized.  Try physically plugging directly into whatever you thought you config'd and see if it shows up that way.

**Comment by cm-ops** [comment]
Is your Windows machine able to resolve that hostname?
